lib/elixir_console_web/views/console_view.ex

```diff
@@ -1,8 +0,0 @@
-defmodule ElixirConsoleWeb.ConsoleView do
-  use ElixirConsoleWeb, :view
-  import Phoenix.LiveView.Helpers, only: [live_component: 3]
-
-  alias ElixirConsoleWeb.ConsoleLive.{CommandInputComponent, HistoryComponent, SidebarComponent}
-
-  def print_prompt, do: "> "
```

Comment:
Glad to see those "view" modules are gone 👏  I can not remember what motivated them (maybe that was the way in early versions or I was plainly wrong 😄) .

Reply:
Yeah probably it wasn't clear how to do it in earlier versions so the most natural way was having views. Probably you could just render a template using a generic view but still the templates had to live in the other directory, far from the components.